megamek/src/megamek/client/ui/swing/tooltip/UnitToolTip.java

```diff
@@ -712,7 +727,26 @@ private static StringBuilder inGameValues(Entity entity, Player localPlayer) {
         // If sensors, display what sensors this unit is using
         if (game.getOptions().booleanOption(OptionsConstants.ADVANCED_TACOPS_SENSORS)
                 || game.getOptions().booleanOption(OptionsConstants.ADVAERORULES_STRATOPS_ADVANCED_SENSORS)) {
-            result.append(addToTT("Sensors", BR, entity.getSensorDesc()));
+            // NOTE: I copied visual range from GeneralInfoMapSet and have not verified its correctness

```

Comment:
If you copied it over, can we possibly get it moved into a separate function instead of duplicating code?

Reply:
Usually a static method within the "Compute" class